wal2json.c

```diff
@@ -758,7 +807,7 @@ pg_decode_change(LogicalDecodingContext *ctx, ReorderBufferTXN *txn,
 	schemaname = get_namespace_name(class_form->relnamespace);
 	tablename = NameStr(class_form->relname);
 
-	OutputPluginPrepareWrite(ctx, true);	
+	init_message(ctx, WAL2JSON_MODIFY_MSG_TYPE, tablename);
```

Comment:
Should we add the table name only or, since we are producing plain text should we turn it into a key value pair stile header `table=TABLENAME` ?

Reply:
No preference, I don't foresee a use case for it (but we didn't foresee this one either, I guess.)

If we need to add more later, I think it's straightforward enough to add them as positional-style values rather than keyword-style.